docs/config.md

```diff
@@ -865,6 +871,7 @@ The following settings are available for Google Cloud Batch:
 : :::{versionadded} 23.11.0-edge
   :::
 : Max number of execution attempts of a job interrupted by a Compute Engine spot reclaim event (default: `5`).
+See also `autoRetryExitCodes`.
```

Comment:
@bentsherman How to link each other these settings?

Reply:
We don't typically do this, but you should be able to put a link target over the config option like `(google-batch-auto-retry-exit-codes)=` and reference it like any other link:
```
: See also: {ref}`google-batch-auto-retry-exit-codes`
```

Reply:
yeah, it would bye much better having automatic links for each option

Reply:
I think Phil and the docs team want to do this at some point. Maybe we can tackle this in the future as we migrate docs